bundles/org.openhab.ui/web/src/components/widgets/plan/oh-plan-marker.vue

```diff
@@ -11,7 +11,8 @@
       <div v-if="config.useTooltipAsLabel" style="white-space: nowrap" :style="tooltipStyle">
         {{ tooltip }}
       </div>
-      <oh-icon v-else-if="config.icon" :icon="config.icon" :color="config.iconColor" :width="config.iconWidth || config.iconSize || 40" :height="config.iconHeight || config.iconSize || 40" :state="config.iconUseState ? state : undefined" />
+      <oh-icon v-else-if="config.icon && config.icon.indexOf('oh:') === 0" :icon="config.icon.split(':')[1]" :width="config.iconWidth || config.iconSize || 40" :height="config.iconHeight || config.iconSize || 40" :state="config.iconUseState ? state : undefined" />
```

Comment:
Revert changes to this file.

Reply:
See comment above on how to convert this correctly...